src/ansys/mapdl/core/mapdl_geometry.py

```diff
@@ -873,6 +891,8 @@ def _select_items(self, items, item_type, sel_type):
         # FLST,5,76,4,ORDE,74
         # FITEM,5,2
         # LSEL, , , ,P51X
+        # if self._item_count(item_type) == 0:
```

Comment:
Still working on this or is it ready to review?

Reply:
I think it is reviewable, but not yet mergeable... I shall have a look at it today again.

Reply:
Consider removing comment or using this section.

Reply:
I'm removing the commented parts I added. I am leaving the things were already there (although commented)